CITATION.cff

```diff
@@ -0,0 +1,32 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: "GLASS: Generator for Large Scale Structure"
+message: >-
+  If you use this software, please cite it using the metadata from this file.
+type: software
+authors:
+  - given-names: Nicolas
+    family-names: Tessore
+  - given-names: Arthur
+    family-names: Loureiro
+  - given-names: Benjamin
+    family-names: Joachimi
+  - given-names: Maximilian
+    family-names: von Wietersheim-Kramsta
+  - given-names: Niall
+    family-names: Jeffrey
```

Comment:
We can add ORCID IDs, affiliations, and contact emails here.

Reply:
I know the academics involved so have suggested their ORCiDs